parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/state/ShardManager.java

```diff
@@ -37,6 +38,7 @@
  *
  * @author Antony Stubbs
  */
+// metrics: number of queues, average queue length
 @Slf4j
 public class ShardManager<K, V> {
```

Comment:
slow work gauge of some sort - how many records are "slow", average wait time, something like that. average size of the slow work collection every time we try to take work.

cc @Ehud-Lev-Forter